### DISCUSSION TITLE
Extra securitylayer for certain sensitive parts of the application

### DISCUSSION THREAD

**Discussion by BOAWEB** [filamentphp/filament#9891] [Help]
Hi all,

I have a question regarding security. Say a user is logged in, and has access to very sensitive data in a filamentphp app.
The user goes afk without logging out and a co-worker comes in, sits behind the computer and as the user is logged in, he can access anything.
Would it be possible to ask for a users' password before accessing certain resources to confirm the user?

If my question is not clear, please let me know.

**Comment by leandrocfe** [comment]
Maybe using a middleware to check this. There is a lock screen plugin: https://filamentphp.com/plugins/marjose123-lockscreen
Maybe this plugin can help you to achieve this.

**Comment by drbyte** [comment]
There's this package for Filament v2: https://github.com/KaanTanis/filament-password-confirmation
And for Filament v3 there's https://github.com/jeffgreco13/filament-breezy which leverages Laravel Breeze in Filament.
